src/globals.h

```diff
@@ -40,7 +40,31 @@
 #define BATTERY_MONITOR BAT_INTERNAL
 #endif
 
-#if defined(LED_INVERTED) && LED_INVERTED
+// If LED_PIN is not defined in "defines.h" take the default pin from "pins_arduino.h" framework.
+// If there is no pin defined for the board, use LED_PIN 255 and disable LED
+#if !defined(LED_PIN)
```

Comment:
We should probably invert these branches, just to make it look a bit neater.

Reply:
ok done